content/300-guides/200-deployment/100-deployment.mdx

```diff
@@ -161,6 +161,13 @@ Prisma Client depends on the [query engine](../../../concepts/components/prisma-
 
 The query engine is implemented in Rust and is used by Prisma in the form of executable binary files. The binary is downloaded when `prisma generate` is called. If the call happens on a different environment when deploying your Prisma-based application to production, you need to ensure that the `binaryTarget` in the Prisma schema is compatible with your production environment as described in [binary targets](/reference/api-reference/prisma-schema-reference#binarytargets-options).
 
+## AWS RDS Proxy
+
+Prisma is compatible with AWS RDS Proxy. However, there is no benefit in using it for connection pooling with Prisma due to the way RDS Proxy pins connections:
+
+> "Your connections to the proxy can enter a state known as pinning. When a connection is pinned, each later transaction uses the same underlying database connection until the session ends. Other client connections also can't reuse that database connection until the session ends. The session ends when the client connection is dropped." ~ [AWS RDS Proxy Docs](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/rds-proxy.html)
+
+Prepared statements and query statements greater than 16 KB cause RDS Proxy to pin the session. Because Prisma uses prepared statements for all queries, you won't see any benefit when using RDS Proxy with Prisma.
```

Comment:
```suggestion
Prepared statements (of any size) or query statements greater than 16 KB cause RDS Proxy to pin the session. Because Prisma uses prepared statements for all queries, you won't see any benefit when using RDS Proxy with Prisma.
```